GNUmakefile

```diff
@@ -168,11 +178,11 @@ website-lint:
 website-lint-fix:
 	@echo "==> Applying automatic website linter fixes..."
 	@misspell -w -source=text website/
-	@docker run -v $(PWD):/markdown 06kellyjac/markdownlint-cli --fix website/docs/
+	@docker run --platform linux/amd64 -v $(PWD):/markdown 06kellyjac/markdownlint-cli --fix website/docs/
```

Comment:
context: `--platform linux/amd64` was added due to compatibility issues with m1 macs. tested on m1 mac @danquack and x86 mac @drewmullen